lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -1007,7 +1007,7 @@ public function AliasIdentificationVariable()
     /**
      * AbstractSchemaName ::= fully_qualified_name | identifier
      *
-     * @return string
+     * @return class-string
```

Comment:
The implementation looks like we don't always get a FQCN here. I'd revert this change.